File: challenges/4-fizz-buzz/python/viniciusenari/fizzbuzz.py
def fizzbuzz(n: int):
    for i in range(1, n + 1):
        print("Fizz" * (i % 3 == 0) + "Buzz" * (i % 5 == 0) or str(i))

if __name__ == '__main__':
    n = int(input("Enter a number: "))
    fizzbuzz(n)

File: challenges/4-fizz-buzz/fsharp/LooserName404/FizzBuzz/FizzBuzz.fs
module FizzBuzz

let (|Fizz|_|) number =
  if number % 3 = 0 then Some ()
  else None

let (|Buzz|_|) number =
  if number % 5 = 0 then Some ()
  else None
  
let fizzBuzz number =
  [ for i in 1..number do
      match i with
      | Fizz & Buzz -> "FizzBuzz"
      | Fizz -> "Fizz"
      | Buzz -> "Buzz"
      | _ -> string i ]
